.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,193 @@
+
+Have an idea but an issue doesn't exist or it's not labelled with `help wanted`? No problem, feel free to kick off the conversation by suggesting a possible solution or by opening a pull request!
+
+If you ever need help, don't hesitate to ask!
+
+# Reporting a bug
+
+Think you've found a bug? Let us know by opening na [issue in our tracker](https://github.com/mortendahl/tf-encrypted/issues) and apply the "bug" label!
+
+### Security Disclosures
+
+Security is a top priority for the project. If you have encounter a security
+issue please responsible disclose it by reaching out to us via
+[contact@dropoutlabs.com](mailto:contact@dropoutlabs.com).
+
+We will work with you to mitigate the security issue and responsibly disclose
+it to anyone using the project.
+
+# Opening a Pull Request
+
+To contribute, [fork](https://help.github.com/articles/fork-a-repo/) `tf-encrypted`, commit your changes, and [open a pull request](https://help.github.com/articles/using-pull-requests/).
+
+You may be asked to make changes to your submission during the review process, we'll work with you on figuring out how to get your pull request ready to be merged and if any changes need to be made.
+
+#### Before submitting
+
+- Run the tests using `make test` which will also lint the code base
+- Test your change thoroughly with unit tests where appropriate
+- Don't forget to add a relevant line to the [CHANGELOG.md](../CHANGELOG.md)!
+- Update the documentation if required inside the `docs` folder and any appropriate doc strings have been added or updated to public functions you changed.
+
+#### Continuous Integration
+
+All pull requests are run against our continuous integration suite on [Circle CI](https://circleci.com/gh/mortendahl/workflows/tf-encrypted). The entire suite must pass before a pull request is accepted. 
+
+#### Body
+
+- Supply examples of command behaviour (command output, daemon logs, etc)
+- Explain why your particular changes were made the way they are
+- Reference the issue your request closes in the body of the PR with `Closes #`
+
+# Documentation
+
+This project uses [Sphinx](http://www.sphinx-doc.org/en/master/) to generate our documentation available on [readthedocs.org](https://tf-encrypted.readthedocs.io/en/latest/index.html).
+
+Whenever a change is made that impacts the behaviour of the API used by
+consumers of this project the corresponding documentation should be updated so
+users always have up to date documentation that reflects the true behaviour of
+the library.
+
+You can build the project locally using the `make docs` command which will
+subsequently output the html version of our docs to your `build` folder. You
+can view the docs after their built using your browser by running the command
+`open build/html/index.html`.
+
+## Typing
```

Comment:
Wonder if we should remove this typing stuff?

Reply:
I think so - but we should do it in all one move.